pkgs/ffigen/lib/src/code_generator/objc_built_in_functions.dart

```diff
@@ -250,7 +250,7 @@ class $name implements ${w.ffiLibraryPrefix}.Finalizable {
   /// Return a pointer to this object.
   $idType get pointer => _id;
 
-  $idType _retainAndReturnId() {
+  $idType retainAndReturnPointer() {
```

Comment:
Generate a dartdoc comment on how to use this.

Reply:
Done